src/java/org/apache/cassandra/utils/JVMStabilityInspector.java

```diff
@@ -58,11 +59,32 @@ private JVMStabilityInspector() {}
      */
     public static void inspectThrowable(Throwable t) throws OutOfMemoryError
     {
-        inspectThrowable(t, true);
+        inspectThrowable(t, null, true, true);
     }
 
+
     public static void inspectThrowable(Throwable t, boolean propagateOutOfMemory) throws OutOfMemoryError
     {
+        inspectThrowable(t, null, propagateOutOfMemory, true);
```

Comment:
this looks mostly used by networking, and the exception thrown is in a different thread... sadly null makes sense given the current usage =(